chance/p5js/Popup-Circles/index.md

```diff
@@ -0,0 +1,9 @@
+---
+layout: p5js
+author: Daniele Maselli
+author-url: https://github.com/DanieleMaselli 
+title: Popup Circles 
+tags: [P5.js, chance, repeat, random, Law of Similarity]
```

Comment:
sorry i though it was just the format wrong....but it is the repeat tag disturbing? or changed it completely